server/db/seed/scrapeRottenTomatoesMovie.js

```diff
@@ -0,0 +1,118 @@
+const puppeteer = require('puppeteer');
+const cheerio = require('cheerio');
+
+var knex = require('./config.js').knex;
+//ROTTEN TOMATOES CRITIC REVIEW SCRAPER
+//NOTE: YOU SHOULD SHUT DOWN ALL MEMORY/CPU HOGS BEFORE ATTEMTPING
+//BE SURE ALL PROPERTIES ARE CORRECTLY SET BEFORE STARTING: node ./server/scrapeRottenTomatoesMovie.js
+//IF MORE THAN ONE MOVIE IS SCRAPED WILL NEED LOGIC TO FIND CRITIC ID WITHOUT DUPES
+//BASE URL: Go to the movie and click all critics in the critic review section to find
+const url = 'https://www.rottentomatoes.com/m/black_panther_2018/reviews/';
+//Starting page number should be 1 unless it crashed
+const startPage = 1;
+//Check the base URL for the maximum number of pages on the reviews page for the movie your attempting and set it here
+const maxPage = 23;
+//Set this with the movie ID listed in the database. Currently set for Blank Panther
+const db_movie_id=284054;

```

Comment:
Was eslint ran?

Reply:
No it wasnt forgot to run it before I pushed up already ran it on my local copy though. I mentioned this to Chen must have forgot to mention this to you.